library/src-3.x/dotty/DottyPredef.scala

```diff
@@ -37,4 +37,12 @@ object DottyPredef {
   }
 
   inline def the[T] given (x: T): x.type = x
+
+  /** Creates a tupled version of this function: instead of N arguments,
+    *  it accepts a single [[scala.Tuple]] argument.
+    */
+  def (f: F) tupled[F, Args <: Tuple, R] given (tf: TupledFunction[F, Args, R]): Args => R = {
```

Comment:
I would repeat the documentation about the `F`, `Args` and `R` types here.

Reply:
Added documentation